### PR TITLE
Fixes #10, Closes #11 : Integrated osm-map and fixed an unknown js bug

### LINKED ISSUE #10: Add `Preview` of the `lat:long` via GMaps
# Current State : 

**_No method to preview the gathered `lat:long`_**

* +1 : **User is Blind**, if the data was correct or not.
* +2 : **User is Blind**, how precise the location was gathered.

# Expected State : 

**There should be !!**

## Are you assigned to this issue ?

**_Heck Ya!!_** :sunglasses:

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #10 

## Changes Done : 

* Added **custom hook** to GMaps :fire: _(bypassed the messy `api` method, with my own workaround.)_ [**~5hrs of researching**]
* Fixed names, Fixed variables
* Enhanced hook, added a hyperlink.
* Fixed `<br>` bug, that caused improper rendering of locations. :100: 
* Added `target` tag to open in separate page, to avoid reloading, going back, or any possible headache.
* embedded **openstreetsmap** :heart: 
* fixed an unknown bug that caused the map to enlarge.
* Minimized the loading times too !

**_Test-Link : https://0x48piraj.github.io/geolocation/_**

## Screenshot : 

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
     <div id="main-div" class="card">
         <label class="label">Click the below button to get Geolocation</label><br><br><br>
-        <input id="button" type="button" value="Get my geolocation" onClick="geolocation()">
+        <input id="button" type="button" value="Get my geolocation" onclick="geolocation()">
         <br><br>
         <!-- The below div will hold the data -->
         <label id="output" class="label"></label>
+        <label id="preview" class="label"></label>
+        <div id="osm-map"></div>
+		<!--<iframe id="frameID" width="200" height="200"></iframe>-->
```
Comment (m1guelpf) [inline index.html]: Why is this commented out? Is it needed?

Comment (0x48piraj) [reply index.html]: No, It was the testing process, as some mentor said that you can use <iframe> to include the map, I tested but it was looking messy, so did via native osm api :blush:

Comment (0x48piraj) [reply index.html]: Should I remove the comment @m1guelpf ?

Comment (m1guelpf) [reply index.html]: It is a bad practice to keep debugging code or code from a previous implementation commented out in production...

Comment (0x48piraj) [reply index.html]: Got it!

Comment (0x48piraj) [reply index.html]: @m1guelpf Done :+1: 
Thanks for telling me this :blush:

**Comment by abishekvashok** [comment]
@0x48piraj good work :)

**Comment by 0x48piraj** [comment]
Thanks a lot @abishekvashok 
This PR became a reality because of you :heart_eyes: 
Not to forget  @m1guelpf and @akshatnitd :100:

**Pull request merged**